submissions/DeamonFire/HTML-CSS-Popup/css/style.css

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
Avoid too many empty(unused) lines in your code.

Reply:
Hi, i removed extra empty lines